pygotham/talks/forms.py

```diff
@@ -13,10 +13,23 @@
 
 
 def duration_query_factory():
-    """Return available :class:`~pygotha.models.Duration` instances."""
+    """Return available :class:`~pygotham.models.Duration` instances."""
```

Comment:
As silly as this sounds, try to avoid making unrelated changes under a pull request. If we had this feature and at some point decide to revert it, this typo will return.